redwood-layout-composeui/src/test/kotlin/app/cash/redwood/layout/composeui/ComposeFlexContainerTest.kt

```diff
@@ -65,27 +69,31 @@ class ComposeFlexContainerTest(
       ),
     ).map {
       // https://github.com/junit-team/junit5/issues/2703
+      @Suppress("UNCHECKED_CAST")
       arrayOf(
         Parameters(
           flexDirection = it[0] as FlexDirection,
-          alignItems = it[1] as AlignItems,
-          justifyContent = it[2] as JustifyContent,
-          padding = it[3] as Padding,
+          items = it[1] as List<String>,
+          alignItems = it[2] as AlignItems,
+          justifyContent = it[3] as JustifyContent,
+          padding = it[4] as Padding,
         ),
       )
     }
 
     class Parameters(
       val flexDirection: FlexDirection,
+      val items: List<String>,
       val alignItems: AlignItems,
       val justifyContent: JustifyContent,
       val padding: Padding,
     ) {
       override fun toString() = "" +
         "FlexDirection.$flexDirection, " +
+        "Items(${items.size}), " +
         "AlignItems.$alignItems, " +
         "JustifyContent.$justifyContent, " +
-        "$padding"
+        "Padding(${padding.start})"
```

Comment:
We could change the `toString` on `Padding` to have this behavior if all four values match

Reply:
Should also do the same if horizontal padding and vertical padding match since there's a `Padding(horizontal, vertical)` overload? Personally I'd vote to keep the standard data class `toString`.

Reply:
Well I'm mostly just guarding against the case where some future traveler adds an asymmetric padding to this test (or copies this helper into another with asymmetric padding) and the test names accidentally collide. I believe paparazzi does nothing to guard against this case. But I also think the case of uniform padding is probably the most common usage.